src/components/donation/Steps/DonateMethods/Stripe/Incrementers.tsx

```diff
@@ -10,7 +10,7 @@ export default function Incrementers({
   code: string;
 }) {
   return (
-    <div className="flex justify-center gap-3">
+    <div className="flex justify-center flex-wrap gap-3">
```

Comment:
wrap incrementors to prevent overflow
<img width="533" alt="image" src="https://github.com/AngelProtocolFinance/angelprotocol-web-app/assets/89639563/4dab2e01-002e-49cd-8c7e-656b93d75832">